soc/arm/st_stm32/common/Kconfig.soc

```diff
@@ -0,0 +1,10 @@
+# Kconfig - ST Microelectronics Common Kconfig
+#
```

Comment:
Ultra nit: I left out the `#` on this line in the cleanup PR.

Reply:
removed.